swap_meet/decor.py

```diff
@@ -1,2 +1,6 @@
-class Decor:
-    pass
+from swap_meet.item import Item
+class Decor(Item):
+    def __init__(self,category = "",condition = 0):
+        super().__init__(category = "Decor",condition = condition)
+    def __str__(self):
```

Comment:
I suggest adding a blank line just for readability

```suggestion
        super().__init__(category = "Decor",condition = condition)

    def __str__(self):
```